airbyte-integrations/connectors/destination-teradata/src/test-integration/java/io/airbyte/integrations/destination/teradata/TeradataDestinationAcceptanceTest.java

```diff
@@ -95,7 +95,7 @@ private List<JsonNode> retrieveRecordsFromTable(final String tableName, final St
   protected void setup(TestDestinationEnv testEnv) {
     final String schemaName = Strings.addRandomSuffix("integration_test_teradata", "_", 5);
     final String createSchemaQuery = String
-        .format(String.format("CREATE DATABASE \"%s\" AS PERM = 1e9 SKEW = 10 PERCENT", schemaName));
+        .format(String.format("CREATE DATABASE \"%s\" AS PERMANENT = 60e6, SPOOL = 60e6 SKEW = 10 PERCENT", schemaName));
```

Comment:
why is this change relevant to this PR?

Reply:
1GB is not required for running integration tests. With this huge memory setting for each database creating while running integration tests, space issues are coming and test cases getting failed. So changed this to limited memory to make integration tests gets success.